### PR TITLE
The Changes recomended by Hayet

### PR DESCRIPTION
**Pull request by canon20paul**
-Have made the changes as you required.
Kindly look through again
**********Thanks***********

### CODE REVIEW

File: src/TasklistFile.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import { update } from "lodash";
```
Comment (laiifuu) [inline src/TasklistFile.js:1]: As stated in the previous review, please remove this import as it is unnecessary.

File: src/index.html [hunk 2]
```diff
@@ -19,6 +19,5 @@ <h1 class="logo">To do List</h1>
             <button type="submit" id="btnDelComplete" >Clear all completed</button>
         </div>
         </main>
-        <footer></footer>
```
Comment (laiifuu) [inline src/index.html:22]: As stated in the previous review, please remove this footer.

File: src/index.js [hunk 3]
```diff
@@ -34,20 +34,20 @@ textInputAdd.addEventListener('keypress',  (e) => {
...
 const taskArray = document.querySelectorAll('.task');
 taskArray.forEach((task) => {
-  task.addEventListener('dblclick', (e) => {
+  task.addEventListener('dblclick', () => {
     task.contentEditable=true;
   });
   task.addEventListener('keypress', (e) => {
     if (e.key === 'Enter') { task.contentEditable = false;
-      for(var i=0; i < TaskList.taskArray.length; i=1+i){
+    for(var i=0; i < TaskList.taskArray.length; i=1+i){
```
Comment (laiifuu) [inline src/index.js:50]: Please use ES6 syntax, and use the `let` keyword instead of `var`.

**Review by laiifuu — CHANGES_REQUESTED** [review]
Hi @canon20paul 👋 
I hope you're doing great 😃 

I've had the chance to have another look at your code. 
There are still issues from the previous review that you might have not paid attention to, as well as other changes concerning your styling. 
You're almost there ! 

- The indentation in your files still needs to be fixed. please adjust it to 2 spaces in all your JS files and your HTML file inside your `src` folder. 
- In your index.js file: 
    - the code here is not readable, please fix indentation and spacing to make it readable. 
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L7-L13
    - Please add some spacing between the following lines for readability purposes: 
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L23-L24
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L28-L29
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L33-L34
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L41-L42
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L58-L59
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L66-L67
    - As stated in the previous review, please remove the unused parameter `task`: 
    https://github.com/canon20paul/To_Do-List-Review/blob/7312906d6a1361732d8b169a9d7c5f53ee25dd7b/src/index.js#L17
    
- If you look at the screenshot below, you can notice that: 
    - The div holding the to do list form is not centered properly. You can fix this by using Flexbox properties on the container holding it and setting that container's width and height to 100%. 
    - Some elements inside the form are overflowing on the right side of their container. You can fix this issue by removing the width property from these elements. 
    - There is some unnecessary white space under the `clear all completed` button, you can fix this by removing the `margin-bottom: 50px;` property on the button. 
    - The `box-shadow` on that same button is also unnecessary, so please remove it. 
    - The black bar at the top is also unnecessary, You can remove it by removing the header tag from the HTML file.
    
localhost_8080_

- #### Please refer to the comments for other issues.

💻 Happy coding 💻

File: src/TasklistFile.js (hunk 1 above)
Comment (canon20paul) [reply src/TasklistFile.js:1]: Kindly I used the Footer and header as Ribbons Asides achieving semantical and accessibility concerns

File: src/index.html (hunk 2 above)
Comment (canon20paul) [reply src/index.html:22]: Kindly I used the Footer and header as Ribbons Asides achieving semantical and accessibility concerns

**Pull request merged**